packages/core/src/internal/utils/global.ts

```diff
@@ -28,7 +28,7 @@ export interface CDSState {
   i18nRegistry: Readonly<Record<string, unknown>>;
   elementRegistry: Readonly<{ [key: string]: any }>;
   iconRegistry: Readonly<Record<string, unknown>>;
-  motionRegistry: Readonly<Record<string, unknown>>;
+  motionRegistry: Readonly<Record<string, any>>;
```

Comment:
My PR includes a fix for this because typescript did not like the `Record` typing here...